Assets/MRTK/SDK/Experimental/Joystick/JoystickController.cs

```diff
@@ -13,13 +13,14 @@ namespace Microsoft.MixedReality.Toolkit.Experimental.Joystick
     /// </summary>
     public class JoystickController : MonoBehaviour
     {
+        [Experimental]
         [SerializeField]
         [Tooltip("The large or small game object that receives manipulation by the joystick.")]
-        private GameObject objectToManipulate = null;
-        public GameObject ObjectToManipulate
+        private GameObject targetObject = null;
```

Comment:
might be good to tag this with [the `[FormerlySerializedAs]` attribute](https://docs.unity3d.com/ScriptReference/Serialization.FormerlySerializedAsAttribute.html).

Reply:
Great idea! Fixed